Remotion/Validation/Core/Validators/ExactLengthValidator.cs

```diff
@@ -15,18 +15,83 @@
 // along with re-motion; if not, see http://www.gnu.org/licenses.
 // 
 using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
 using JetBrains.Annotations;
+using Remotion.FunctionalProgramming;
+using Remotion.Utilities;
 using Remotion.Validation.Implementation;
+using Remotion.Validation.Results;
 
 namespace Remotion.Validation.Validators
 {
-  public class ExactLengthValidator : LengthValidator
+  public class ExactLengthValidator : ILengthValidator
   {
+    public int Min { get; }
+
+    public int? Max { get; }
+
+    public string ErrorMessage { get; }
+
+    public ValidationMessage ValidationMessage { get; }
+
     public ExactLengthValidator (int length, [NotNull] ValidationMessage validationMessage)
```

Comment:
Ah, my mistake. I overlooked that the Min and Max values are public on the interface. I still think you should not have the overload of the ctor since the private one is never called from anywhere but the public one.